apps/etterlatte-behandling/src/main/resources/db/migration/V152__trigger_lytte_sakendringer.sql

```diff
@@ -0,0 +1,43 @@
+-- Basert på https://www.cybertec-postgresql.com/en/tracking-changes-in-postgresql/
+CREATE SCHEMA logging;
+
+CREATE TABLE logging.t_history
+(
+    id         serial,
+    timestamp  timestamp DEFAULT now(),
+    schemaname text,
+    tabname    text,
+    operation  text,
+    who        text      DEFAULT current_user,
```

Comment:
Vil ikke dette være databasebrukeren? Vi vil vel egentlig ha det på "Systembruker" / "Saksbehandler"-nivå. Så jeg tror vi må ha noe som kjører business-logikk i dao / service her tenker jeg

Reply:
> Vil ikke dette være databasebrukeren? Vi vil vel egentlig ha det på "Systembruker" / "Saksbehandler"-nivå. Så jeg tror vi må ha noe som kjører business-logikk i dao / service her tenker jeg

Tenkte på dette som ein dealbreaker, men lurer på om vi kan omgå det med ei `sist endra av`-kolonne. Trur det vil funke for både oppdater og legg til, men kanskje ikkje trivielt for slett. Hmm.

Reply:
Det slo meg faktisk at det kan være nyttig å ha med `current_user` selv om det vil være databasebrukeren. 

Om vi legger til logging av bruker som har endret vil vi også kunne se hvilken utvikler som har gjort endringer direkte. På den måten kan vi kanskje også droppe alt rælet med migreringsscript for enkle updates siden vi da både får blame og historikk 🤷